### PR TITLE
v7-fixlogout: refactor(plugin-credentials): Fixed issues with logging out

### PR DESCRIPTION
**Pull request by saurjai3**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "Bernie Zang <nzang@cisco.com>",
     "Vagou Zhou <vagouz@cisco.com>"
   ],
-  "main": "src/index.js",
+  "main": "packages/recipe-private-web-client/dist/index.js",
```
Comment (ianwremmel) [inline package.json]: Please don't do this. The web client should reference `@ciscospark/recipe-private-web-client`, not `spark-js-sdk`

File: README.md [hunk 2]
```diff
@@ -70,6 +70,39 @@ This is mostly useful for the the example app(s), but also comes in handy when d
 ```bash
 PACKAGE=PACKAGENAME npm run grunt:package -- serve
 ```
+### Running eslint on a single package
```
Comment (ianwremmel) [inline README.md]: "Run"

File: packages/plugin-credentials/src/credentials/common.js [hunk 3]
```diff
@@ -201,14 +208,14 @@ export default {
     return Reflect.apply(SparkPlugin.prototype.initialize, this, ...args);
   },
 
-  @waitForValue(`@`)
+  @waitForValue(`isLoggingOut`)
```
Comment (ianwremmel) [inline packages/plugin-credentials/src/credentials/common.js]: Logout should not be called until `@` is loaded and `isLoggingOut` should not be persisted (make it a session variable prevent it from being persisted).

**Comment by saurjai3** [comment]
changes no more needed.

**Pull request closed without merging**